src/components/calculator.js

```diff
@@ -1,50 +1,42 @@
-        <button type="button" className={yellowButton} onClick={this.clickHandler}>-</button>
-        <button type="button" className={uniformStyles} onClick={this.clickHandler}>1</button>
-        <button type="button" className={uniformStyles} onClick={this.clickHandler}>2</button>
-        <button type="button" className={uniformStyles} onClick={this.clickHandler}>3</button>
-        <button type="button" className={yellowButton} onClick={this.clickHandler}>+</button>
-        <button type="button" className="col-6 div height border d-flex justify-content-center align-items-center" onClick={this.clickHandler}>0</button>
-        <button type="button" className={uniformStyles} onClick={this.clickHandler}>.</button>
-        <button type="button" className={yellowButton} onClick={this.clickHandler}>=</button>
-      </div>
-    );
-  }
-}
+  const uniformStyles = 'col-3 div height border d-flex justify-content-center align-items-center';
+  const yellowButton = `${uniformStyles} bg-warning`;
+  return (
+    <div className="row border w-50">
+      <div className="col-12 div height border d-flex justify-content-end align-items-end bg-secondary text-light">{next || total || '0'}</div>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>AC</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>+/-</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>%</button>
+      <button type="button" className={yellowButton} onClick={clickHandler}>÷</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>7</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>8</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>9</button>
+      <button type="button" className={yellowButton} onClick={clickHandler}>x</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>4</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>5</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>6</button>
+      <button type="button" className={yellowButton} onClick={clickHandler}>-</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>1</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>2</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>3</button>
+      <button type="button" className={yellowButton} onClick={clickHandler}>+</button>
+      <button type="button" className="col-6 div height border d-flex justify-content-center align-items-center" onClick={clickHandler}>0</button>
+      <button type="button" className={uniformStyles} onClick={clickHandler}>.</button>
+      <button type="button" className={yellowButton} onClick={clickHandler}>=</button>
```

Comment:
- [ ] **[Optional] Please consider improving the code for the creation of the buttons as it seems redundant.**

Each line of code is creating the same component. Only the props and the value for each component are different. I think some sort of collection might fix the problem. Probably a hash, or also an array?